bin/protoc-gen-ruby_gapic

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env ruby
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+$LOAD_PATH << ::File.expand_path('../../lib', __FILE__)
```

Comment:
Better `unshift` rather than `<<` to put the lib directory at the head of the path.